changelog/3224.feature

```diff
@@ -0,0 +1 @@
+Allowed test selection by file:line_number. For example, you can run ``pytest tests/test_foo.py:5`` and Pytest will find a nearest test.
```

Comment:
We also need to add some docs to it, I suggest we add them to https://docs.pytest.org/en/latest/usage.html after *Run tests by node ids*.

Reply:
Fixed